src/TuDa.CIMS.Web/Components/AssetList.razor.cs

```diff
@@ -0,0 +1,123 @@
+using Microsoft.AspNetCore.Components;
+using TuDa.CIMS.Shared.Entities;
+
+
+namespace TuDa.CIMS.Web.Components;
+
+public partial class AssetListPartialClass
+{
+    /// <summary>
+    /// List of items to be shown
+    /// </summary>
+    [Parameter]
+    public List<AssetItem> AssetItems { get; set; } = s_sampleValues;
+
+    private static readonly List<AssetItem> s_sampleValues =
+    [
+        new Chemical
+        {
+            Id = new Guid(),
+            Note = "Temperatur: -78,5\u00b0 C",
+            Room = new Room { Id = new Guid(), Name = "Audimax" },
+            Name = "Trockeneis",
+            Cas = "124-38-9",
+            ItemNumber = "1845",
+            Shop = "Eisladen",
+            Hazards = [new Hazard { Id = new Guid(), Name = "kalt", ImagePath = "" }],
+            Unit = "kg"
+        },
+        new Chemical
+        {
+            Id = new Guid(),
+            Note = "AKA Alkohol",
+            Room = new Room { Id = new Guid(), Name = "Bistro Athene" },
+            Name = "Ethanol",
+            Cas = "64-17-5",
+            ItemNumber = "1170",
+            Shop = "Taverne",
+            Hazards = [new Hazard { Id = new Guid(), Name = "macht süchtig", ImagePath = "" }],
+            Unit = "l"
+        },
+        new Chemical
+        {
+            Id = new Guid(),
+            Note = "Feststoff",
+            Room = new Room { Id = new Guid(), Name = "Bosch-Hörsaal" },
+            Name = "Natriumhydroxid",
+            Cas = "1310-73-2",
+            ItemNumber = "1823",
+            Shop = "ALDI",
+            Hazards = [new Hazard { Id = new Guid(), Name = "ätzend", ImagePath = "" }],
+            Unit = "kg"
+        },
+        new Consumable
+        {
+            Id = new Guid(),
+            Note = "zerbrechlich",
+            Room = new Room { Id = new Guid(), Name = "Glashaus/10" },
+            Name = "Reagenzglas",
+            ItemNumber = "2770/30",
+            Shop = "Mediamarkt",
+            Manufacturer = "Glasmacher",
+            SerialNumber = "12345678910"
+        }
+    ];
+
+    /// <summary>
+    /// Returns type of the given item.
+    /// </summary>
+    private static string GetItemType(AssetItem assetItem)
+    {
+        return assetItem is Chemical ? "Chemikalie" : "Verbrauchsmaterial";
+    }
```

Comment:
🤔 🌱  Maybe we should use `switch` here too, so we can have a default value. This can currently not happen, but maybe in the future.
```suggestion
    private static string GetItemType(AssetItem assetItem) =>
        assetItem switch
        {
            Chemical => "Chemikalie",
            Consumable => "Verbrauchsmaterial",
            _ => "-",
        };
```